pdr_backend/trader/trader_agent.py

```diff
@@ -124,12 +124,12 @@ def _process_block_at_feed(self, addr: str, timestamp: int, tries: int = 0):
 
         print(f"Got {prediction}.")
 
-        self._get_trader(feed, prediction)
+        self._trade(feed, prediction)
```

Comment:
"trade() is gonna be hard to distinguish in some places. Now that I understand the intent a bit better, I recommend calling it "do_trade()".